ground/src/main/java/com/google/android/ground/repository/SurveyRepository.kt

```diff
@@ -132,4 +132,12 @@ constructor(
   ): @Cold(terminates = false) Flowable<List<Mutation>> {
     return localDataStore.getMutationsOnceAndStream(survey)
   }
+
+  suspend fun deleteSurvey(surveyId: String) {
```

Comment:
Could we perhaps name this `removeOfflineSurvey()` or similar? 

From the repository client's standpoint, surveys are either available offline, or not, and the fact that the survey is deleted from the local db is an implementation detail.

Also, this would make it clear that we're not actually deleting the source-of-truth survey on the server.